docs/pages/toolchain/sozo/common-options/configurations.md

```diff
@@ -0,0 +1,47 @@
+# Account Options
+
+This module provides options for configuring an account in the `sozo` command.
+
+## Configuration Priorities
+
+The configuration system follows the following priority order:
+
+1. Command-line arguments: If options are specified via command-line arguments, they have the highest priority.
+2. Environment variables: Environment variables provide a way to configure options globally.
+3. Configuration in `Scarb.toml`: Any configuration found within the `Scarb.toml` file will be considered if options are not found in the first two steps.
+
+## Account Options
+
+- `account_address`: StarkNet account address. Can be specified via command-line arguments or environment variables.
```

Comment:
```suggestion
- `account_address`: Starknet account address. Can be specified via command-line arguments or environment variables.
```

Reply:
We already described how the command line and env variables work, we may not have to specify again how the argument can be specified?